README.md

```diff
@@ -41,6 +42,15 @@ NOTE: Please make sure you configure the appropriate container runtime based on
 
 ## Usages
 
+### Use GenAI Microservices Connector(GMC) to deploy and adjust GenAIExamples
+
+Follow [GMC README](https://github.com/opea-project/GenAIInfra/blob/main/microservices-connector/README.md)
+to install GMC into your kubernetes cluster. GMC provides sample config for GenAIExamples under `microservices-connector/config/samples`. Select the example workflow you want to deploy, deploy the example (e.g. DocSum) using `kubectl`:
+
+```shell
```

Comment:
not safe to say this .. because some of the examples need environment variable set up, possible creation of namespace etc. So just delegate to its readme file.

Reply:
modified using above readme from GenAIExamples